extras/vagrantExample/src/main/vagrant/readme.md

```diff
@@ -41,6 +41,18 @@ Once Vagrant is installed, starting up the Rya-Example-Box is fairly straightfor
 
 1. Download the [Rya-Example-Box Vagrantfile] to the custom directory.  Note that it must be named `Vagrantfile` with no extension.
 
+1. Optionally copy built Rya libraries from your project into the vagrant directory.  If not, it will attempt to download from the public repository.  The following files can be copied or linked (ln -s file destination).

```

Comment:
vagrant directory is the custom directory mentioned above? If so, I would keep the name consistent and refer to it as the "custom directory" instead of "vagrant directory"
Maybe be more explicit with where the built Rya libraries are coming from. For example, 
"Optionally, if you downloaded and built Rya, copy built Rya libraries from your project into the custom directory.  If not, Vagrant will attempt to download from the public repository when building the VM.  The following files can be copied or linked (ln -s file destination)."